src/scss/blocks/nova-blocks/hero/_common.scss

```diff
@@ -1,7 +1,7 @@
 .novablocks-hero {
 
     &[class] a {
-        --theme-links-color: var(--novablocks-hero-text-color);
+        --theme-links-color: var(--nb-hero-text-color);
```

Comment:
Is this variable still relevant after we've integrated Color Signal?

Reply:
Probably not.